logic/src/commonTest/kotlin/com/wire/kalium/logic/feature/message/SendTextMessageUseCaseTest.kt

```diff
@@ -0,0 +1,229 @@
+//
+//        given(messageRepository)
+//            .suspendFunction(messageRepository::updateMessage)
+//            .whenInvokedWith(anything())
+//            .thenReturn(
+//                Either.Left(CoreFailure.Unknown(IllegalArgumentException()))
+//            )
+//
+//        given(messageSender)
+//            .suspendFunction(messageSender::trySendingOutgoingMessageById)
+//            .whenInvokedWith(anything(), anything())
+//            .thenReturn(Either.Left(CoreFailure.Unknown(IllegalArgumentException())))
+//        //when
+//        val result = sendTextMessageUseCase(ConversationId("test", "test"), "text")
+//
+//        //then
+//        verify(messageRepository)
+//            .suspendFunction(messageRepository::persistMessage)
+//            .with(anything())
+//            .wasInvoked(Times(1))
+//
+//        verify(messageSender)
+//            .suspendFunction(messageSender::trySendingOutgoingMessageById)
+//            .with(anything(), anything())
+//            .wasInvoked(Times(1))
+//
+//        assertIs<SendTextMessageResult.Failure>(result)
+//    }
+//
+//    @Test
+//    fun givenSendingTextMessageSucceeds_whenSendingTextMessage_SuccessIsPropagatedCorrectly() = runTest {
+//        //given
+//        given(userRepository)
+//            .suspendFunction(userRepository::getSelfUser)
+//            .whenInvoked()
+//            .thenReturn(flowOf(TestUser.SELF))
+//
+//        given(clientRepository)
+//            .suspendFunction(clientRepository::currentClientId)
+//            .whenInvoked()
+//            .thenReturn(Either.Right(PlainId("test")))
+//
+//        given(messageRepository)
+//            .suspendFunction(messageRepository::persistMessage)
+//            .whenInvokedWith(anything())
+//            .thenReturn(Either.Right(Unit))
+//
+//        given(messageRepository)
+//            .suspendFunction(messageRepository::updateMessage)
+//            .whenInvokedWith(anything())
+//            .thenReturn(Either.Right(Unit))
+//
+//        given(messageSender)
+//            .suspendFunction(messageSender::trySendingOutgoingMessageById)
+//            .whenInvokedWith(anything(), anything())
+//            .thenReturn(Either.Right(Unit))
+//        //when
+//        val result = sendTextMessageUseCase(ConversationId("test", "test"), "text")
+//
+//        //then
+//        verify(messageRepository)
+//            .suspendFunction(messageRepository::persistMessage)
+//            .with(anything())
+//            .wasInvoked(Times(1))
+//
+//        verify(messageRepository)
+//            .suspendFunction(messageRepository::updateMessage)
+//            .with(anything())
+//            .wasInvoked(Times(1))
+//
+//        verify(messageSender)
+//            .suspendFunction(messageSender::trySendingOutgoingMessageById)
+//            .with(anything(), anything())
+//            .wasInvoked(Times(1))
+//
+//        assertIs<SendTextMessageResult.Success>(result)
+//    }
+//
+//}
```

Comment:
Is this intentional? Why commenting the tests out?

Reply:
this is fro mthe previous take on this, those tests are not valid anymore , will remove it, thanks